README.md

```diff
@@ -116,6 +116,20 @@ eg.
 python3 predict.py --path_to_dataset data.hdf5 --path_to_model TIMED.h5 --path_to_output .
 ```
 
+**The output files include:**
+ 
+i. `{MODEL_NAME}.csv` , Contains probabilities for different amino acids at each position of the protein as predicted by the selected model.
+
+ii. `{MODEL_NAME}.fasta` , Protein sequences generated by selecting the amino acid with the highest probability (ArgMax) at each position as indicated by the probability predictions in the `{MODEL_NAME}.csv` file.
+
+iii. `{MODEL_NAME}.txt` , A file listing the the protein/ chain ID and the number of its residues for amino acid sequences found in {MODEL_NAME}.fasta file.  
+
+iv. `dataset.fasta` , Fasta sequences fed into the model.
+
+v. `datasetmap.txt` , Fasta sequences fed into the model.
```

Comment:
The sequence does not go into the model. It's the fasta sequence of the structure. 

datasetmap does not contain sequences.